### ISSUE TITLE
Recherche: The "title" attribute value is in English on the French page

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#161]
**Page:** Recherche
**URL:** https://rsams.dev.cloud.openplus.ca/fr/recherche/site?search_api_fulltext=test&f%5B0%5D=public%3A215
**User Agent:** Firefox (v99)
**Language:** French

**Issue Description:** The "title" attribute value is in English for the "Effacer tous les filtres" button.

![image](https://user-images.githubusercontent.com/12256067/162261290-4ea47ae2-582d-4e85-9eed-794b5534129c.png)

**Steps to reproduce:**

1. Go to the above URL
2. Click on any filters (e.g. from public) from the right columns
3. Hover over the "Effacer tous les filtres" button displayed below the search feature at the beginning of the page

**Expected results:** The "title" attribute value should be provided in the language of the page.

**Comment by 40999221** [comment]
Issued fixed, the title attribute in accordance with the page language. But the purpose of the title is to give supplementary information to the link before user use the link. if link text and title are the same then screen reader will read the information twice. So in this case further updates needed by removing the title from the link.

![image](https://github.com/user-attachments/assets/7436a2f5-f6bf-43b8-9c8e-43d2a6e59f74)

**Comment by mmiron-HC** [comment]
@40999221  Did you apply any of the filters that appear on the right? if you do, you will get the additional text.

**Comment by zimw7** [comment]
Issue fixed. The 'Effacer tous les filtres' button now has the correct French title attribute. 
However, the title is redundant for screen readers. It should be removed to avoid reading the same information twice.
![Snap2](https://github.com/user-attachments/assets/5c71225c-f7f5-4a54-9966-d9abc414d81b)

**Comment by YuxuanXie01** [comment]
Not sure. The title attribute for the "Effacer tous les filtres" button is currently the same as the button text, "Effacer tous les filtres", which causes redundancy and leads NVDA to announce it twice. In contrast, on the English page, the button text "Clear all" is complemented by a title attribute "Remove all filters," which provides additional context. 
Further, the "title" attribute value displays as a tooltip when user hovers over the link. However, it is not keyboard accessible.
I think adjusting the title attribute on the French page to either offer additional context or removing it if unnecessary to prevent repetitive announcements by screen readers.
![image](https://github.com/user-attachments/assets/bf56ffbb-00fa-405e-a443-2c6b0e8847e7)
![image](https://github.com/user-attachments/assets/47265278-49ba-4323-90ae-1703fa82b5af)

**Comment by mmiron-HC** [comment]
@YuxuanXie01, @zimw7, @40999221   Agreed that the title is not needed here as it is the same as the visible text. Good point that not all tooltips are also accessible to keyboard users.  For now, we can close this ticket as the original issue has been fixed,
